templates/server-route.yaml

```diff
@@ -1,33 +1,33 @@
-{{- if .Values.global.openshift }}
-{{- if ne .mode "external" }}
-{{- if .Values.server.route.enabled -}}
-{{- $serviceName := include "vault.fullname" . -}}
-{{- if eq .mode "ha" }}
-{{- $serviceName = printf "%s-%s" $serviceName "active" -}}
-{{- end }}
-kind: Route
-apiVersion: route.openshift.io/v1
-metadata:
-  name: {{ template "vault.fullname" . }}
-  labels:
-    helm.sh/chart: {{ include "vault.chart" . }}
-    app.kubernetes.io/name: {{ include "vault.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    {{- with .Values.server.route.labels }}
-      {{- toYaml . | nindent 4 }}
-    {{- end }}
-  {{- template "vault.route.annotations" . }}
-spec:
-  host: {{ .Values.server.route.host }}
-  to:
-    kind: Service
-    name: {{ $serviceName }}
-    weight: 100
-  port:
-    targetPort: 8200
-  tls:
-    termination: passthrough
-{{- end }}
-{{- end }}
-{{- end }}
+{{- if .Values.global.openshift }}
```

Comment:
Seems to be quite a bit of whitespace change, dos2unix :)

Reply:
It does look strange, doesn't it?

Reply:
Looks like we had DOS line feeds. Seems to have been the only file affected.

```
$ git ls-files -z | xargs -0 file|grep CRLF
templates/server-route.yaml:                                ASCII text, with CRLF line terminators
```